### ISSUE TITLE
Crash when opening a folder on 1.2.2

### ISSUE THREAD

**Issue by Salnikoff** [files-community/Files#3525]
I'm trying to open any folder and program is crashing...
back to v1.2

**Comment by yaichenbaum** [comment]
@Salnikoff Can you upload the logs?

**Comment by gave92** [comment]
Hopefully same as #3504

**Comment by Salnikoff** [comment]
[files.txt](https://github.com/files-community/Files/files/5961111/files.txt)

**Comment by Salnikoff** [comment]
https://user-images.githubusercontent.com/25884425/107564100-f4e77800-6bf2-11eb-907d-9f233d45b25a.mp4

**Comment by gumbarros** [comment]
Same

**Comment by gave92** [comment]
@Salnikoff does this happen also if the folder layout mode is Grid View?

**Comment by okey2k** [comment]
same issue after upgrading to 1.2.2 while 1.2 is all good.

crashing every time with every folder.

**Comment by okey2k** [comment]
https://user-images.githubusercontent.com/32120698/107569992-383ce900-6be9-11eb-921b-2dc10b198e93.mp4

**Comment by Salnikoff** [comment]
https://user-images.githubusercontent.com/25884425/107570598-d08f9980-6bfa-11eb-9026-33dc69639e22.mp4

**Comment by okey2k** [comment]
I find the issue is caused by double click, if I switch to single click to open will temporarily mitigate the problem.
![Screenshot 2021-02-10 215018](https://user-images.githubusercontent.com/32120698/107570724-37588700-6bea-11eb-9fa9-88c214dcb75b.png)

**Comment by Salnikoff** [comment]
> I find the issue is caused by double click, if I switch to single click to open will temporarily mitigate the problem.
> ![Screenshot 2021-02-10 215018](https://user-images.githubusercontent.com/32120698/107570724-37588700-6bea-11eb-9fa9-88c214dcb75b.png)

same

**Comment by rusti4** [comment]
I think they mentioned the crash would be fixed in 1.2.3.0 or something. I had this issue with 1.2.0.0, but it persists in 1.2.2.0. Obviously for some its created the problem.

**Comment by KaseyMahoney** [comment]
I'm having the same issue with crashing. Downloaded for the first time today from the windows store and no matter what, it crashed after attempting to go go into a new folder. I'm going to try and see if the single click setting does the trick.

https://user-images.githubusercontent.com/63125709/107603137-b0eb7780-6be0-11eb-87d7-a063d7cd091d.mp4

2021-02-10 20:12:03.2440|INFO|.ctor|Drive added: C:\, Fixed|
2021-02-10 20:12:03.2939|INFO|.ctor|Drive added: D:\, Fixed|
2021-02-10 20:12:03.3308|INFO|.ctor|Drive added: E:\, Fixed|
2021-02-10 20:12:04.6074|INFO|.ctor|Running on ARM: False|
2021-02-10 20:12:04.6074|INFO|.ctor|Running on ARM: **False|**

**Comment by KaseyMahoney** [comment]
On a positive note, switching my preferences to 'open items with a single click' has stopped it from crashing.

**Comment by gumbarros** [comment]
'Open items with a single click' mitigate the problem for me too :)

**Comment by hez2010** [comment]
I cannot repro it on my machine even opening a large folder (such as `C:\Windows\WinSxS`).

Translation for logs provided above:
```log
Level Date and Time Source Event ID Task Category
Error 10/02/2021 22:34:51 Application Error 1000 (100) "Faulting application name: Files.exe, version: 1.0.0.0, timestamp: 0x6021a170
Faulting module name: Windows.UI.Xaml.dll, version: 10.0.19041.789, timestamp: 0x29b8b079
Exception code: 0xc0000005
Error offset: 0x000000000073a306
Faulting Process ID: 0x2adc
Faulting application start time: 0x01d6ffe3c9779687
Faulting Application Path: C:\Program Files\WindowsApps\49306atecsolution.FilesUWP_1.2.2.0_x64__et10x9a9vyk8t\Files.exe
Faulting module path: C:\Windows\System32\Windows.UI.Xaml.dll
Report ID: 7a4c46b9-3cc4-4e12-85bb-0f4cd23899d1
Faulting package full name: 49306atecsolution.FilesUWP_1.2.2.0_x64__et10x9a9vyk8t
The application code associated with the failing package is App " 
```

@gave92 Seems that there was an access violation occurred in `Windows.UI.Xaml.dll` (Windows 10.0.19041.789) and it's an exception from unmanaged code, is it possible related to Windows 10 20H1? (I'm running dev insider preview, so I don't have Windows 10.0.19041.789 environment to test it)

**Comment by gave92** [comment]
I think the access violation has been fixed by #3511. Are you testing with master or store version?

**Comment by hez2010** [comment]
> @hez2010 I think the access violation has been fixed by #3511. Are you testing with master or store version?

store version

**Comment by gave92** [comment]
@hez2010 pity :) I used to experience this issue only on few folders, for some users this issue happens almost everywhere. I'd really like to confirm whether #3511 fixes this..

**Comment by rusti4** [comment]
@gave92  @hez2010 

I recently updated to 21H1 and now files is working.

Edition	Windows 10 Pro
Version	21H1
Installed on	‎28/‎10/‎2020
OS build	19043.844
Experience	Windows Feature Experience Pack 120.2212.2020.0

**Comment by rusti4** [comment]
oh it just froze and crashed

**Comment by gave92** [comment]
Fixed in 1.3 The remaining startup issue is #3531
